web/package.json

```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e --webdriverUpdate=false",
+    "postinstall": "node_modules/.bin/webdriver-manager update --gecko false --standalone false --versions.chrome=84.0.4147.89"
```

Comment:
Sorry I forgot to mention one question. Should we add the ChromeDriver version to `protractor.conf.js`  file?

Reply:
Is there a way to do that in there?

Reply:
I was looking at the link you sent earlier. https://stackoverflow.com/a/58644510

Reply:
I don't think we need to do that because that points to driver that was already downloaded. In our case we only download one.

Reply:
I see, that makes sense.